module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
```

Comment:
Why do you need utility include in the header? Generally you prefer to have as little as possible includes in the header in favour of cpp includes, because you do not want to waste time including library in place where you do not need it.